homeassistant/components/sensor/xiaomi_aqara.py

```diff
@@ -77,5 +77,5 @@ def parse_data(self, data):
             return False
         elif self._data_key == 'pressure' and value == 0:
             return False
-        self._state = round(value, 2)
+        self._state = round(value, 1)
```

Comment:
Is on purpose?

Reply:
Yes the PR was already approved. I completely forgot about it. My apologies.
https://github.com/home-assistant/home-assistant/pull/11673